backend/application/syson-sysml-import/src/test/java/org/eclipse/syson/sysml/ASTTransformerTest.java

```diff
@@ -38,6 +46,19 @@
  */
 public class ASTTransformerTest {
 
+    // All standard libraries
+    private static List<Resource> libResource;
```

Comment:
Please rename `libResource ` to `standardLibraries` and remove the comment

Reply:
Done